proto/qdrant/cloud/iam/v1/iam.proto

```diff
@@ -175,6 +175,19 @@ service IAMService {
       body: "*"
     };
   }
+  // Logs out the authenticated user.
+  // Required permissions:
+  // - None (authenticated only)
+  rpc Logout(LogoutRequest) returns (LogoutResponse) {
+    // permissions
+    option (common.v1.permissions) = "";
+    // custom account-id expression
+    option (qdrant.cloud.common.v1.account_id_expression) = "";
+    // custom supported-actor-types expression. (Requires to be authenticated as a user)
+    option (qdrant.cloud.common.v1.supported_actor_types) = ACTOR_TYPE_USER;
+    // gRPC Gateway REST call
+    option (google.api.http) = {get: "/api/iam/v1/users/me/logout"};
```

Comment:
I think using `post` for this would be better.

Reply:
Fully agree, missed that